Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -3,6 +3,7 @@
   from gyb_syntax_support.kinds import lowercase_first_word
   from gyb_syntax_support.kinds import SYNTAX_BASE_KINDS
   from gyb_syntax_support.kinds import syntax_buildable_child_type, syntax_buildable_default_init_value
+  from gyb_syntax_support.protocolsMap import PROTOCOL_CONFORMANCES_MAP
```

Comment:
Since you asked in the other PR regarding naming: If we only model conformances using the `ExpressibleAs` protocols, I would suggest: `SYNTAX_BUILDABLE_EXPRESSIBLE_AS_CONFORMANCES`.